pyvista/plotting/lookup_table.py

```diff
@@ -748,7 +750,7 @@ def apply_opacity(self, opacity, interpolate: bool = True, kind: str = 'quadrati
         """
         if isinstance(opacity, (float, int)):
             if opacity < 0 or opacity > 1:
-                raise ValueError(f'Opacity must bet between 0 and 1, got {opacity}')
+                raise ValueError(f'Opacity must be between 0 and 1, got {opacity}')
             self.values[:, -1] = opacity * 255
         elif len(opacity) == self.n_values:
```

Comment:
Right below this line (I can't annotate it, alas) we have
```py
            self.values[:, -1] = np.array(opacity, copy=False).copy()
```
Why is this not just
```py
            self.values[:, -1] = np.array(opacity)
```
Am I missing something here?

Reply:
You're not missing anything. This is why we have reviewers.